scripts/globals/events/login_campaign_data.lua

```diff
@@ -0,0 +1,463 @@
+-- This file is called by login_campaign.lua and require()'s no file, it should not be require()'d by any
+-- other lua scripts, which should instead require() login_campaign.lua directly.
+
+-- items table can only hold up to 20 items per option.
+tpz.events.loginCampaign.prizes =
+{
+    [2019] =
+    {
+        [11] =
+        {
+            [1] =
+            {
+                ["price"] = 10,
+                ["items"] =
+                {
+                     1126, -- Beastmen's Seal
+                     1127, -- Kindred's Seal
+                     2955, -- Kindred's Crest
+                     2956, -- High Kindred's Crest
+                     2957, -- Sacred Kindred's Crest
+                     1857, -- Cordial Invite
+                     2306, -- Martial Ball Invite
+                     5364, -- Training Grounds Key
+                     2487, -- Mercenary Camp Entry Slip
+                     5741, -- Flask Of Pest Repellent
+                     3557, -- Athena Orb
+                     5113, -- Cracked Nut
+                     3541, -- Seasoning Stone
+                     3543, -- Fossilized Fang
+                     3542, -- Fossilized Bone
+                     5724, -- Pungent Powder
+                     6535, -- Pungent Powder II
+                     9890, -- Tarazacum Orb
+                },
+            },
+
+            [5] =
```

Comment:
These options can be simplified if the value you're passing in is `<<2` prior.  1..25 simplifies to 0..6:
![image](https://user-images.githubusercontent.com/12447174/110208978-06b9e500-7e58-11eb-9dc9-f792ef15a5de.png)

Reply:
^